backend/btrixcloud/models.py

```diff
@@ -1072,16 +1258,59 @@ class CollIn(BaseModel):
 
     name: str = Field(..., min_length=1)
     description: Optional[str] = None
+    caption: Optional[str] = None
     crawlIds: Optional[List[str]] = []
 
-    isPublic: bool = False
+    access: CollAccessType = CollAccessType.PRIVATE
+
+
+# ============================================================================
+class CollOut(BaseMongoModel):
+    """Collection output model with annotations."""
+
+    name: str
+    oid: UUID
+    description: Optional[str] = None
+    caption: Optional[str] = None
+    modified: Optional[datetime] = None
+
+    crawlCount: Optional[int] = 0
+    pageCount: Optional[int] = 0
+    totalSize: Optional[int] = 0
+
+    dateEarliest: Optional[datetime] = None
+    dateLatest: Optional[datetime] = None
+
+    # Sorted by count, descending
+    tags: Optional[List[str]] = []
+
+    access: CollAccessType = CollAccessType.PRIVATE
+
+    homeUrl: Optional[AnyHttpUrl] = None
+    homeUrlTs: Optional[datetime] = None
+    homeUrlPageId: Optional[UUID] = None
+
+    resources: List[CrawlFileOut] = []
+    thumbnail: Optional[ImageFileOut] = None
 
 
 # ============================================================================
-class CollOut(Collection):
+class PublicCollOut(BaseMongoModel):
     """Collection output model with annotations."""
 
+    name: str
+    oid: UUID
+    description: Optional[str] = None
+    caption: Optional[str] = None
+
+    dateEarliest: Optional[datetime] = None
+    dateLatest: Optional[datetime] = None
+
+    homeUrl: Optional[AnyHttpUrl] = None
+    homeUrlTs: Optional[datetime] = None
+
     resources: List[CrawlFileOut] = []
```

Comment:
I missed this, but [from the mockups](https://private-user-images.githubusercontent.com/4672952/390177135-415d1b27-9b32-4d22-92a4-646b40bbd6c9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyODQzNjQsIm5iZiI6MTczMzI4NDA2NCwicGF0aCI6Ii80NjcyOTUyLzM5MDE3NzEzNS00MTVkMWIyNy05YjMyLTRkMjItOTJhNC02NDZiNDBiYmQ2YzkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMDRUMDM0NzQ0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDNjNGE3Y2JmNzRhOWY2MTZkYTY4NWU0ZjA1ZGNmZjI5NzQxYzM0N2VkODU3ZDg3ZDZmMDdhYWUxOTkyMGYxMyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.kyngjs_kr6sSojLrhz6H-PEKKPtRHh8t9Mj8bzROWY0) we'll also need to return `totalSize`, `pageCount`, and `crawlCount`.

Reply:
Added!